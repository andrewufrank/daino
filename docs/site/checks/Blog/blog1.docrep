DocrepJSON {yam = Object (fromList [("fileEntries",Array []),("style",Null),("indexPage",Bool False),("link",String ""),("bibliography",Null),("lang",String "DLenglish"),("date",String "2020-06-18"),("keywords",String "Blog"),("author",String ""),("dirEntries",Array []),("abstract",String "Ein Blog ohne Sinn auf Deutsch"),("title",String "Mein erster Blog"),("fn",String "/home/frank/Workspace11/ssg/docs/site/dough/Blog/blog1.md"),("publish",Null)]), pan = Pandoc (Meta {unMeta = fromList []}) [Header 1 ("ein-erster-abschnitt",[],[]) [Str "Ein",Space,Str "erster",Space,Str "Abschnitt"],Para [Str "Ein",Space,Str "Blog",Space,Str "ohne",Space,Str "Sinn",Space,Str "und",Space,Str "dem",Space,Str "einzigen",Space,Str "Zweck,",Space,Str "zu",Space,Str "testen,",Space,Str "wie",Space,Str "ein",Space,Str "Blog",Space,Str "in",SoftBreak,Str "ein",Space,Str "PDF",Space,Str "umgewandelt",Space,Str "wird."],Header 1 ("dies-ist-der-zweite-abschnitt",[],[]) [Str "Dies",Space,Str "ist",Space,Str "der",Space,Str "zweite",Space,Str "Abschnitt"],Para [Str "und",Space,Str "auch",Space,Str "ein",Space,Str "bischen",Space,Str "text."],Header 2 ("mit-einer-unterabschnitt",[],[]) [Str "mit",Space,Str "einer",Space,Str "unterabschnitt"],Para [Str "hier."],Para [Str "das",Space,Str "waers.",Space,Str "es",Space,Str "fehlt",Space,Str "Referenzen,",Space,Str "listen",Space,Str "und",Space,Str "aehnliches"]]}