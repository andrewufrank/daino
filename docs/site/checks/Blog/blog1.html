<!DOCTYPE html>
<!-- the master page for the pandoc templating mechanism
	needs page-title, page-title-postfix
		author date and keywords for indexing
    contentHtml, beforeContent, afterContent to wrap html content  
	uses style.css as default, other can be loaded with key css 
	-->
<html lang="">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />


  <style type="text/css">
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
	      	q { quotes: "“" "”" "‘" "’"; }
	  </style>

	
		  <link rel="stylesheet" href="/templates/static/style.css" />
		  <link rel="stylesheet" href="/templates/static/menu.css" />
	
              <meta name="dcterms.date" content="2019-01-01" />
        
	<title></title>
	
        <style>
            body {
            margin: 0;
            font-family: Arial, Helvetica, sans-serif;
            }

            .hero-image {
            background-image: url('/templates/img/symmetricGeras2.jpg');
            background-color: white;
            height: 200px;
            background-position: center;
            background-repeat: no-repeat;
            background-size: cover;
            position: relative;
            }

            .hero-text {
            transform: translate(-50%, -50%);
            color: white;
            text-shadow: 2px 2px 8px black;
            font-weight: bold;
            vertical-align: top;
            }
        </style>
    </head>

    <body>
    <!-- AF - my template for pandoc page (from the default) page3.dtpl -->
        

    <section class="hero">
        <div class="hero-image">
            
                <h1>siteNameExample</h1>
                <h3>siteByLineExample</h3>
        <div/>
    </section>
    <section class="header">

            <section class="menu">
		<ul>
            					<li><a href=/Blog/index.html>Blog</a></li>
            						<li><a href=/PublicationList/index.html>Publications</a></li>
            						<li><a href=/SSGdesign/index.html>SSG Documentation</a></li>
            				</ul>
    </section>
    <section>
         
    </section>

    <section class=menu2>
		<ul>
            					<li>
				    <a href=.html><em> </em></a>
				    		     
				    <span class=tiny> 2000-01-01 00:00:00 UTC 
            .html zero
                    </span>
                    
    			</li>
            				</ul>
    </section>

        
<!-- new content section -->
    <section class=header>
    <h1 id="ein-erster-abschnitt">Ein erster Abschnitt</h1>
<p>Ein Blog ohne Sinn und dem einzigen Zweck, zu testen, wie ein Blog in ein PDF umgewandelt wird.</p>
<h1 id="dies-ist-der-zweite-abschnitt">Dies ist der zweite Abschnitt</h1>
<p>und auch ein bischen text.</p>
<h2 id="mit-einer-unterabschnitt">mit einer unterabschnitt</h2>
<p>hier.</p>
<p>das waers. es fehlt Referenzen, listen und aehnliches</p>
    </section>


        <p class=tiny>
        Produced with SGG 3.0 on 2000-01-01 00:00:00 UTC from Path Rel File Blog/blog1.md with master4.dtpl.
        </p>

            </body>


</html>
