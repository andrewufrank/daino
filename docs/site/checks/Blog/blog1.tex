\documentclass[a4paper,10pt]{scrbook}
\usepackage{fontspec}
\usepackage[ngerman]{babel}
\usepackage{graphicx}
\usepackage{makeidx}
\usepackage{natbib}
\makeindex
\usepackage[colorlinks]{hyperref}
\providecommand{\tightlist}{%
\setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\begin{document}

\hypertarget{ein-erster-abschnitt}{%
\section{Ein erster Abschnitt}\label{ein-erster-abschnitt}}

Ein Blog ohne Sinn und dem einzigen Zweck, zu testen, wie ein Blog in
ein PDF umgewandelt wird.

\hypertarget{dies-ist-der-zweite-abschnitt}{%
\section{Dies ist der zweite
Abschnitt}\label{dies-ist-der-zweite-abschnitt}}

und auch ein bischen text.

\hypertarget{mit-einer-unterabschnitt}{%
\subsection{mit einer unterabschnitt}\label{mit-einer-unterabschnitt}}

hier.

das waers. es fehlt Referenzen, listen und aehnliches

\printindex
\end{document}
