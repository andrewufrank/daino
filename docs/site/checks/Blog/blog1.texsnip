TexSnip {snipyam = Object (fromList [("fileEntries",Array []),("style",Null),("indexPage",Bool False),("link",String ""),("bibliography",Null),("lang",String "DLenglish"),("date",String "2020-06-18"),("keywords",String "Blog"),("author",String ""),("dirEntries",Array []),("abstract",String "Ein Blog ohne Sinn auf Deutsch"),("title",String "Mein erster Blog"),("fn",String "/home/frank/Workspace11/ssg/docs/site/dough/Blog/blog1.md"),("publish",Null)]), unTexSnip = "\\hypertarget{ein-erster-abschnitt}{%\n\\section{Ein erster Abschnitt}\\label{ein-erster-abschnitt}}\n\nEin Blog ohne Sinn und dem einzigen Zweck, zu testen, wie ein Blog in\nein PDF umgewandelt wird.\n\n\\hypertarget{dies-ist-der-zweite-abschnitt}{%\n\\section{Dies ist der zweite\nAbschnitt}\\label{dies-ist-der-zweite-abschnitt}}\n\nund auch ein bischen text.\n\n\\hypertarget{mit-einer-unterabschnitt}{%\n\\subsection{mit einer unterabschnitt}\\label{mit-einer-unterabschnitt}}\n\nhier.\n\ndas waers. es fehlt Referenzen, listen und aehnliches"}