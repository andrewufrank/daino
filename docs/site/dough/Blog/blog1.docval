{
    "masterTemplate": "master4.dtpl",
    "localhostPort": 3000,
    "today": "2000-01-01 00:00:00 UTC",
    "testVal": "testValPage3yaml",
    "settings": {
        "sitename": "siteNameExample",
        "banner": "/templates/img/symmetricGeras2.jpg",
        "byline": "siteByLineExample"
    },
    "afterContent": "</section>",
    "beforeContent": "<section class=header>",
    "menu": [
        {
            "text": "Blog",
            "link": "/Blog/index.html"
        },
        {
            "text": "Publications",
            "link": "/PublicationList/index.html"
        },
        {
            "text": "SSG Documentation",
            "link": "/SSGdesign/index.html"
        }
    ],
    "css": [
        "/templates/static/style.css",
        "/templates/static/menu.css"
    ],
    "storage": {
        "doughDir": "/home/frank/Workspace11/ssg/docs/site/dough",
        "reportFile": "/home/frank/SSGreport.txt",
        "uploadServer": "sgg.gerastree.at",
        "themeDir": "/home/frank/Workspace11/ssg/theme",
        "bakedDir": "/home/frank/Workspace11/ssg/docs/site/baked",
        "testDir": "/home/frank/.SSG"
    },
    "quotes": true,
    "menu2": [
        {
            "text2": "",
            "title2": "",
            "abstract2": "",
            "isIndex": false,
            "author2": "",
            "link2": ".html",
            "publish2": "zero",
            "date2": "2000-01-01 00:00:00 UTC"
        }
    ],
    "date": "2019-01-01",
    "contentHtml": "<h1 id=\"ein-erster-abschnitt\">Ein erster Abschnitt</h1>\n<p>Ein Blog ohne Sinn und dem einzigen Zweck, zu testen, wie ein Blog in ein PDF umgewandelt wird.</p>\n<h1 id=\"dies-ist-der-zweite-abschnitt\">Dies ist der zweite Abschnitt</h1>\n<p>und auch ein bischen text.</p>\n<h2 id=\"mit-einer-unterabschnitt\">mit einer unterabschnitt</h2>\n<p>hier.</p>\n<p>das waers. es fehlt Referenzen, listen und aehnliches</p>",
    "ssgversion": "3.0",
    "author": null,
    "filename": "Path Rel File Blog/blog1.md"
}