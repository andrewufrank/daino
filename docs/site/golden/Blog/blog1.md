---
title: Mein erster Blog
keywords: Blog
abstract: Ein Blog ohne Sinn auf Deutsch
date: 2020-06-18
---

# Ein erster Abschnitt
Ein Blog ohne Sinn und dem einzigen Zweck, zu testen, wie ein Blog in 
ein PDF umgewandelt wird.

# Dies ist der zweite Abschnitt
und auch ein bischen text.

## mit einer unterabschnitt
hier.

das waers. es fehlt Referenzen, listen und aehnliches
