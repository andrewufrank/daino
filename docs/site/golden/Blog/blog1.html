<!DOCTYPE html>
<!-- the master page for the pandoc templating mechanism
	needs page-title, page-title-postfix
		author date and keywords for indexing
    contentHtml, beforeContent, afterContent to wrap htmlcontent  
	uses style.css as default, other can be loaded with key css 
	-->
<html lang="DLenglish">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />


  <style type="text/css">
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
	  </style>

	
	
          <meta name="author" content="FILL AUTHOR" />
              <meta name="dcterms.date" content="2020-06-18" />
              <meta name="keywords" content="Blog" />
    
	<title></title>
	
        <style>
            body {
            margin: 0;
            font-family: Arial, Helvetica, sans-serif;
            }

            .hero-image {
            background-image: url('');
            background-color: white;
            height: 200px;
            background-position: center;
            background-repeat: no-repeat;
            background-size: cover;
            position: relative;
            }

            .hero-text {
            transform: translate(-50%, -50%);
            color: white;
            text-shadow: 2px 2px 8px black;
            font-weight: bold;
            vertical-align: top;
            }
        </style>
    </head>

    <body>
    <!-- AF - my template for pandoc page (from the default) page3.dtpl -->
        

    <section class="hero">
        <div class="hero-image">
            
                <h1></h1>
                <h3></h3>
        <div/>
    </section>
    <section class="header">

            <section class="menu">
		<ul>
            			</ul>
    </section>
    <section>
                <header>
        <h1 class="title">Mein erster Blog</h1>
        
                <p class="author">FILL AUTHOR</p>
                </header>
         
    </section>

    <section class=menu2>
		<ul>
            					<li>
				    <a href=Blog/blog1.HTML><em>Mein erster Blog </em></a>
				    <br>Ein Blog ohne Sinn auf Deutsch		     
				    <span class=tiny>FILL AUTHOR "2020-06-18" 
            Blog/blog1.HTML Nothing
                    </span>
                    
    			</li>
            				</ul>
    </section>

        
<!-- blog text content section -->
    
    abstractEin Blog ohne Sinn auf Deutsch 
    content-from-panrep-blocks<h1 id="ein-erster-abschnitt">Ein erster Abschnitt</h1>
<p>Ein Blog ohne Sinn und dem einzigen Zweck, zu testen, wie ein Blog in ein PDF umgewandelt wird.</p>
<h1 id="dies-ist-der-zweite-abschnitt">Dies ist der zweite Abschnitt</h1>
<p>und auch ein bischen text.</p>
<h2 id="mit-einer-unterabschnitt">mit einer unterabschnitt</h2>
<p>hier.</p>
<p>das waers. es fehlt Referenzen, listen und aehnliches</p>
    


        <p class=tiny>
        Produced with SGG  on  from  with master4.dtpl.
        </p>

            </body>


</html>
